docs/samples/other-charts/bubble.md

```diff
@@ -1,27 +1,49 @@
 # Bubble
 
 ```js chart-editor
+// <block:setup:1>
+const DATA_COUNT = 7;
+const NUMBER_CFG = {count: DATA_COUNT, rmin: 5, rmax: 15, min: 0, max: 100};
+
+const data = {
+  datasets: [
+    {
+      label: 'Dataset 1',
+      data: Utils.bubbles(NUMBER_CFG),
+      borderColor: Utils.CHART_COLORS.red,
+      backgroundColor: Utils.transparentize(Utils.CHART_COLORS.red, 0.5),
+    },
+    {
+      label: 'Dataset 2',
+      data: Utils.bubbles(NUMBER_CFG),
+      borderColor: Utils.CHART_COLORS.orange,
+      backgroundColor: Utils.transparentize(Utils.CHART_COLORS.orange, 0.5),
+    }
+  ]
+};
+// </block:setup>
+
 // <block:actions:2>
 const actions = [
   {
     name: 'Randomize',
     handler(chart) {
       chart.data.datasets.forEach(dataset => {
-        dataset.data = Utils.bubbles({count: chart.data.labels.length, rmin: 5, rmax: 15, min: 0, max: 100});
+        dataset.data = Utils.bubbles({count: DATA_COUNT, rmin: 5, rmax: 15, min: 0, max: 100});
       });
       chart.update();
     }
   },
   {
     name: 'Add Dataset',
     handler(chart) {
-      const data = chart.data;
```

Comment:
keep this `data`, so its consistent with all the other sampels

Reply:
> 'data' is already declared in the upper scope on line 5 column 7 . eslint([no-shadow](https://eslint.org/docs/rules/no-shadow))

Reply:
They are both block scoped so it shouldn't matter and they are for the user also in different tabs.

CI way of running lint never fails on it but you run it in another way, idk why it fails but its weird.

Might be worth to see if we can not let the rule count for the docs then or disable it since how I see it, it does not seem to work correctly.

If it does not work it should be renamed in all the samples but my preference goes to having it like it was